pkgs/build-support/setup-hooks/build-single-xml-catalog.sh

```diff
@@ -0,0 +1,21 @@
+# Creates a single XML catalog that references the catalogs found by
+# findXMLCatalogs.
+# Useful for situations where only one catalog is expected.
+buildSingleXMLCatalog() {
```

Comment:
Probably better if this takes a single argument which is the output file produced.